src/Microsoft.Framework.Runtime.Roslyn/RoslynProjectReference.cs

```diff
@@ -152,12 +169,12 @@ public IDiagnosticResult EmitAssembly(string outputPath)
 
                 var sw = Stopwatch.StartNew();
 
-                EmitResult result = null;
+                EmitResult emitResult = null;
```

Comment:
Ehh, it doesn't look like it needs to be initialized regardless. Just a code cleanup comment.